tox.ini

```diff
@@ -33,6 +33,7 @@ deps =
     ordereddict
     simplejson
     unittest2
+    wheel==0.29.0
```

Comment:
would it make a difference to install this before the other requirements? Just throwing out guesses here.